.github/workflows/ci-workflow.yaml

```
@@ -0,0 +1,23 @@
+name: CI Workflow 
+on: [push, pull_request]
+jobs:
+  build:
+    name: build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [11.x]
```

```suggestion
        node-version: [10.x]
```

based on https://github.com/freeCodeCamp/chapter/blob/bea3dcc1c8174daa3dc942b59380e1cf8c67b373/docker-compose.yml#L4

Recommend reworking the below steps in run in a `node:10-alpine` Docker container as well.